content/authors/georgiana-dolocan/_index.md

```diff
@@ -49,3 +49,5 @@ user_groups:
 - Open Engineering Team
 - Staff
 ---
+
+Software Engineer irreversibly in love with open source. A JupyterHub team member, focusing on infrastructure and community growth. Previously JupyterHub Contributor in Residence and Outreachy intern through an internship that supports diversity in open source and free software.
```

Comment:
_"irreversibly in love with open source"_ made me smile wide :heart_eyes:

Reply:
Awww :heart:  :heart_eyes: :heart: I'm so happy :heart: This was the first sentence that came to my mind when trying to describe the me-work relationship.